### PR TITLE
Sys.env() can now read environment variable with templating

### PR DESCRIPTION
**Pull request by jrask**

### CODE REVIEW

File: docs/enrich.rst [hunk 1]
```diff
@@ -57,9 +57,25 @@ It is also possible to use templates for reading system properties and environme
...
+The approach above is only possible when using lumbermill functions with String types which does not always work. However,
+using a number or boolean or a completely separate instance of something is common.
+
+.. code-block:: groovy
+
+    new MyThirdPartyThing(
+        env ('{HOST || localhost').string(),
+        env ('{PORT || 8080').number(),
+        env ('{CREATE || false').bool())
+
```
Comment (somcsel) [inline docs/enrich.rst]: These 3 strings should end with a }

**Review by somcsel — COMMENTED** [review]
sorry for the incredibly long delay, I saw the notification during the summer but then forgot about it until I came back to this git on Friday.

Changes look good!

File: docs/enrich.rst (hunk 1 above)
Comment (jrask) [reply docs/enrich.rst]: +1

**Comment by sel** [comment]
:+1:
Lgtm!

**Review by sel — APPROVED** [review]

**Pull request merged**